README.md

```diff
@@ -12,16 +12,22 @@ Add to your serverless.yml:
 
   custom:
     assets:
-      bucket: my-bucket
-      files:
-        - source: ../assets/
-          globs: **/*.css
-        - source: ../app/
-          globs:
-            - **/*.js
-            - **/*.map
+       - bucket: my-bucket
```

Comment:
Shouldn't this same format change be made to the ACL example below?

Either that or comment that for a single bucket....